app/uk/gov/hmrc/nationalinsurancerecord/connectors/DesConnector.scala

```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.nationalinsurancerecord.connectors
 
 import com.google.inject.Inject
 import play.api.Logger
```

Comment:
Do we still need this import if we are now extending Logger

Reply:
removed